client/components/empty-content/README.md

```diff
@@ -44,7 +44,9 @@ function render() {
 | `actionURL`             | `string`        | `null`                                   | `href` value for the primary action button.                                                                                                   |
 | `actionCallback`        | `function`      | `null`                                   | `onClick` value for the primary action button.                                                                                                |
 | `actionTarget`          | `string`        | `null`                                   | If omitted, no target attribute is specified.                                                                                                 |
-| `actionHoverCallback`\* | `bool`          | `0`                                      | Indicates activity while a background action is being performed.                                                                              |
+| `actionHoverCallback`\* | `bool`          | `0`                                      | Indicates activity while a background action is being performed.                                                                             |
+| `actionDisabled`        | `bool`          | `null`                                   | Disables the button.                                                                                                     |
+| `actionRef`             | `function`      | `null`                                   | Adds a ref to the button.                 |
```

Comment:
It looks like the linter picked up some whitespace issues in this file?

Reply:
Thanks, I missed that!